apps/src/code-studio/pd/workshop_enrollment/WorkshopEnrollmentCelebrationDialog.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import Button from '@cdo/apps/templates/Button';
```

Comment:
nit: can you use the component library button if possible?

Reply:
Yuuup. I fall into this trap when I'm using existing code as a guide- I made two tickets to go back and update the two files I was copying from! I'd also like to have a broader discussion about how to avoid this moving forward. @kelbyhawn and I discussed some options and are curious about a custom lint rule, as well as a larger move to update components that have since been replaced. https://codedotorg.atlassian.net/browse/ACQ-2055 and https://codedotorg.atlassian.net/browse/ACQ-2056

Reply:
Updated! And put those tickets into next sprint